core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -1264,16 +1272,24 @@ protected void initLocationIndex() {
     }
 
     private boolean isCHPrepared() {
-        return "true".equals(ghStorage.getProperties().get(CH.PREPARE + "done"))
-                // remove old property in >0.9
-                || "true".equals(ghStorage.getProperties().get("prepare.done"));
+        return "true".equals(ghStorage.getProperties().get(CH.PREPARE + "done"));
     }
 
-    private boolean isLMPrepared() {
-        return "true".equals(ghStorage.getProperties().get(Landmark.PREPARE + "done"));
+    private String getProfileHash(String profile) {
+        return ghStorage.getProperties().get("graph.profiles." + profile + ".hash");
+    }
+
+    private void setProfileHash(String profile, int version) {
```

Comment:
Can we either call it hash or version (but not both :)) ?

Reply:
Haha, sure. I even had a third option in the code before ;)